wire-ios/Wire-iOS/Sources/AppStateCalculator.swift

```diff
@@ -99,30 +99,29 @@ final class AppStateCalculator {
     private var hasEnteredForeground: Bool = false
 
     // MARK: - Private Implementation
-    private func transition(to appState: AppState,
-                            completion: (() -> Void)? = nil) {
+    private func transition(
+        to appState: AppState,
+        completion: @escaping () -> Void = {}
+    ) {
         guard hasEnteredForeground  else {
             pendingAppState = appState
-            completion?()
-            return
+            return completion()
```

Comment:
🤔 is there a difference in execution order?

Reply:
I reverted this change.